# Example configuration:
# - velocity limits are around a 10% above the physical limits
# - acceleration limits are just low enough to avoid jerking

# Mandatory parameters
speed_lim_v_x: 1.0
speed_lim_v_y: 1.0
speed_lim_w: 5.4

accel_lim_v_x: 0.3
accel_lim_v_y: 0.3
accel_lim_w: 1.5

# Optional parameters
frequency: 20.0
decel_factor: 10.0

# Robot velocity feedback type:
#  0 - none
#  1 - odometry
#  2 - end robot commands
robot_feedback: 1
